src/style.css

```
@@ -1 +1,62 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+
+.stars--1 :nth-child(1) {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 :nth-child(-n + 2) {
```

```suggestion
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
... {
```
Consider using commas to group selectors and avoid repeating code